src/test/resources/students/TestUser100.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Are these files generated every time? If so, they should be in the autoGenerated folder? Or are they there for examples? In which case they should go under the authoring folder...

Reply:
moved to test/resources/author/students/...